docs/source/conf.py

```diff
@@ -44,8 +45,14 @@
 }
 autoclass_content = "both"
 
-nbsphinx_allow_errors = True
-nbsphinx_execute = "never"
+nb_execution_mode = "auto"
+nbsphinx_allow_errors = False
+nbsphinx_custom_formats = {
+    ".pct.py": ["jupytext.reads", {"fmt": "py:percent"}],
+}
+nbsphinx_execute_arguments = ["--InlineBackend.figure_formats={'svg', 'pdf'}"]
+# If window is narrower than this, input/output prompts are on separate lines:
+nbsphinx_responsive_width = "700px"
```

Comment:
Could you shortly describe for me what these nbsphinx commands are doing? Thanks!

Reply:
Line 53 renders figures as `svg` files. This is preferable to `png` as the images are vectors so will scale to large displays better. The `pdf` option is there for people exporting the docs to LaTeX or pdfs.
Line 55 will ensure code chunks render nicely on mobiles/tables.

Reply:
Great, thanks!